app/transformers/rural-payments-portal/customer.js

```diff
@@ -1,56 +1,52 @@
 export const ruralPaymentsPortalCustomerTransformer = data => {
   return {
-    info: {
-      name: {
-        title: data.title,
-        otherTitle: data.otherTitle,
-        first: data.firstName,
-        middle: data.middleName,
-        last: data.lastName
-      },
-      dateOfBirth: data.dateOfBirth,
-      phone: {
-        mobile: data.mobile,
-        landline: data.landline,
-        fax: data.fax
-      },
-      email: {
-        address: data.email,
-        validated: data.emailValidated,
-        doNotContact: data.doNotContact
-      },
-      address: {
-        pafOrganisationName: data.address.pafOrganisationName,
-        buildingNumberRange: data.address.buildingNumberRange,
-        buildingName: data.address.buildingName,
-        flatName: data.address.flatName,
-        street: data.address.street,
-        city: data.address.city,
-        county: data.address.county,
-        postalCode: data.address.postalCode,
-        country: data.address.country,
-        uprn: data.address.uprn,
-        dependentLocality: data.address.dependentLocality,
-        doubleDependentLocality: data.address.doubleDependentLocality,
-        typeId: data.address.addressTypeId
-      },
-      status: {
-        locked: data.locked,
-        confirmed: data.confirmed,
-        deactivated: data.deactivated
-      }
+    name: {
+      title: data.title,
+      otherTitle: data.otherTitle,
+      first: data.firstName,
+      middle: data.middleName,
+      last: data.lastName
     },
-    crn: data.customerReferenceNumber,
-    customerId: `${data.id}`
+    dateOfBirth: data.dateOfBirth,
+    phone: {
+      mobile: data.mobile,
+      landline: data.landline,
+      fax: data.fax
+    },
+    email: {
+      address: data.email,
+      validated: data.emailValidated,
+      doNotContact: data.doNotContact
+    },
+    address: {
+      pafOrganisationName: data.address.pafOrganisationName,
+      buildingNumberRange: data.address.buildingNumberRange,
+      buildingName: data.address.buildingName,
+      flatName: data.address.flatName,
+      street: data.address.street,
+      city: data.address.city,
+      county: data.address.county,
+      postalCode: data.address.postalCode,
+      country: data.address.country,
+      uprn: data.address.uprn,
+      dependentLocality: data.address.dependentLocality,
+      doubleDependentLocality: data.address.doubleDependentLocality,
+      typeId: data.address.addressTypeId
+    },
+    status: {
+      locked: data.locked,
+      confirmed: data.confirmed,
+      deactivated: data.deactivated
+    }
   }
 }
 
 export function transformPersonRolesToCustomerAuthorisedBusinessesRoles (customerId, personRoles) {
-  return personRoles.filter(({ personId }) => personId.toString() === customerId).map(({ role }) => role)
+  return personRoles.filter(({ personId }) => personId === customerId).map(({ role }) => role)
```

Comment:
The value returns from the mock API is a number. This worked before because the data had already been transformed into a string: https://github.com/DEFRA/fcp-data-access-layer-api/pull/43/files/6f768b412fb7bab2129d09d4d96562d846e09c06#diff-8c65b74830a7b6a0dd4edd71b1c58b3903b1b71b2779a416a830da47e7a0443aL44